llvm/test/CodeGen/PowerPC/aix-tls-gd-double.ll

```diff
@@ -156,10 +156,11 @@ define void @storesTIInit(double %Val) #0 {
 ; SMALL32:       # %bb.0: # %entry
 ; SMALL32-NEXT:    mflr 0
 ; SMALL32-NEXT:    stwu 1, -32(1)
-; SMALL32-NEXT:    lwz 3, L..C4(2) # target-flags(ppc-lo) @TIInit
-; SMALL32-NEXT:    lwz 4, L..C5(2) # target-flags(ppc-tlsgd) @TIInit
 ; SMALL32-NEXT:    stw 0, 40(1)
-; SMALL32-NEXT:    bla .__tls_get_addr[PR]
+; SMALL32-NEXT:    lwz 6, L..C4(2) # target-flags(ppc-tlsld) @TIInit
```

Comment:
These cases changed because `TargetMachine::getTLSModel()` decides TIInit model is TLSModel::LocalDynamic. Before this patch, this case is handled by the default GD model in `PPCTargetLowering::LowerGlobalTLSAddressAIX()`. Now with LD support, they changed model.

Reply:
Better to post an NFC patch to explicitly specify these thread local variables to be `general-dynamic` similar to what you do in `llvm/test/CodeGen/PowerPC/aix-tls-local-dynamic.ll`, so they are not affected by this PR.

Reply:
I'm afraid `generaldynamic` is not a valid explicit mode according to document: https://llvm.org/docs/LangRef.html
```
A variable may be defined as thread_local, which means that it will not be shared by threads (each thread will have a separated copy of the variable). Not all targets support thread-local variables. Optionally, a TLS model may be specified:

localdynamic

    For variables that are only used within the current shared library.
initialexec

    For variables in modules that will not be loaded dynamically.
localexec

    For variables defined in the executable and only used within it.

If no explicit model is given, the “general dynamic” model is used.
```

Reply:
How about using `llc --thread-model=...`?

Reply:
How about adding `-relocation-model=pic` to RUN lines?

Reply:
I just notice the change here is related to an `internal` symbol, so the change is expected after local-dynamic is enabled.